disnake/stage_instance.py

```diff
@@ -135,6 +143,17 @@ def is_public(self) -> bool:
         )
         return self.privacy_level is StagePrivacyLevel.public
 
+    @property
+    def guild_scheduled_event(self) -> Optional[GuildScheduledEvent]:
+        """Optional[:class:`GuildScheduledEvent`]: The stage instance's scheduled event.
+
+        This is only set if this stage instance has an associated scheduled event,
+        and requires that event to be cached.
+        """
```

Comment:
Should this have a comment that it will only resolve if the intent is enabled?

Reply:
7631af88461f783f25f8e847b21578c726777cf1